tests/unit/compareUserInListTest.ts

```diff
@@ -0,0 +1,24 @@
+import compareUserInList from '@libs/compareUserInList';
+
+describe('compareUserInList', () => {
+    it('should compare the weight if the weight is difference', () => {
+        const first = {displayName: 'John Doe', weight: 1, accountID: 1};
+        const second = {displayName: 'Jane Doe', weight: 2, accountID: 2};
+        expect(compareUserInList(first, second)).toBe(-1);
+    });
+
+    it('Should compare the displayName if the weight is the same', () => {
```

Comment:
Make everything consistent. One is lowercase, another is uppercase in "should".
Also typo: difference > different